charts/oam-core-resources/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "oam-core-resources.fullname" . }}
+  labels:
+    {{- include "oam-core-resources.labels" . | nindent 4 }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "oam-core-resources.selectorLabels" . | nindent 6 }}
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        {{- include "oam-core-resources.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "oam-core-resources.serviceAccountName" . }}
+      securityContext:
+          {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Release.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          command:
+            - /manager
+          args:
+            - "--metrics-addr=127.0.0.1:8080"
+            - "--enable-leader-election"
+          image: {{ .Values.image.repository }}:{{ .Chart.AppVersion }}
```

Comment:
we usually don't use `.Chart.AppVersion` as a image tag. You could define a tag in values file like `.Values.image.tag`

Reply:
Another easier way: just make it:
```
image: {{ .Values.image.image }}
```